backend/routes/profile.py

```
@@ -0,0 +1,109 @@
+from typing import List
+from fastapi import APIRouter, Depends, HTTPException, UploadFile, File, Form
+from sqlalchemy.orm import Session
+import models.user as models
+from database.session import SessionLocal
+from models.user import User  # ✅ Correct model import
+from schemas.user import UserResponse  # ✅ Correct schema import
+from api.v1.endpoints.auth import get_current_user  # Authentication dependency
+import os
+import uuid
```

Ruff (0.8.2)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)